src/tests/test_claim_detector.py

```
@@ -0,0 +1,400 @@
+        assert valid_claim.text == "The Earth is round"
+        assert valid_claim.context == "This is a valid context"
+
+    @pytest.mark.integration
+    @pytest.mark.asyncio
+    async def test_real_agent_integration(self):
+        """Integration test with real agent."""
+        # Test factual claim
+        factual_claims = await process_claims("The study found that 75% of participants showed improvement.")
+        assert len(factual_claims) > 0
+        assert all(isinstance(claim, Claim) for claim in factual_claims)
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)